django/contrib/admin/helpers.py

```diff
@@ -83,8 +83,8 @@ def media(self):
             extra = '' if settings.DEBUG else '.min'
             js = [
                 'vendor/jquery/jquery%s.js' % extra,
-                'jquery.init.js',
                 'collapse%s.js' % extra,
+                'jquery.init.js',
```

Comment:
How does this work? `collapse.js` requires `django.jquery` -- wouldn't moving to the end break this?